locale/en.json

```diff
@@ -128,6 +129,7 @@
 		"identity-block.forgot-password-instruction-submitted": "If there is an account associated with your email address, you’ll receive a link in your inbox to reset your password.",
 		"identity-block.forgot-password-submit": "Send",
 		"identity-block.log-in": "Log In",
+		"identity-block.log-in-natural": "Log in",
```

Comment:
@LauraPinilla Okay, then is the version with a capital "In" still used anywhere? I do not see it any of the figma diagrams? It would be better to update `identity-block.log-in` instead of making a new key that needs to be tracked in Lokalise. It's not much but there is a cost for adding more of these keys.